sklearn/cluster/_birch.py

```diff
@@ -694,7 +694,13 @@ def predict(self, X):
 
     def _predict(self, X):
         """Predict data using the ``centroids_`` of subclusters."""
-        kwargs = {"Y_norm_squared": self._subcluster_norms}
+
+        # The extra `Y_norm_squared` argument for the back-end
```

Comment:
Did the old backend support `Y_norm_squared`?

Reply:
Yes.

Reply:
Yes it does, but your comment indicates that this is not clear: I am going to clarify it.